metrics/notifier.go

```diff
@@ -45,23 +49,23 @@ func (metrics *NotifierMetrics) UpdateFetchNotificationsDurationMs(fetchNotifica
 	metrics.fetchNotificationsDurationMs.Update(time.Since(fetchNotificationsStartTime).Milliseconds())
 }
 
-// MarkSendersDroppedNotifications marks metrics as 1 by contactType for dropped notifications.
-func (metrics *NotifierMetrics) MarkSendersDroppedNotifications(contactType string) {
-	if metric, found := metrics.SendersDroppedNotifications.GetRegisteredMeter(contactType); found {
+// MarkContactDroppedNotifications marks metrics as 1 by contactType for dropped notifications.
+func (metrics *NotifierMetrics) MarkContactDroppedNotifications(contactType string) {
+	if metric, found := metrics.ContactsDroppedNotifications.GetRegisteredMeter(contactType); found {
```

Comment:
Супер микро придирка (можешь не исправлять даже), но наверно с точки зрения го идиоматичнее было бы делать 

```suggestion
	if metric, ok := metrics.ContactsDroppedNotifications.GetRegisteredMeter(contactType); ok {
```

Reply:
Решил не трогать, т.к. по смыслу тут подходит